packages/amplify-provider-awscloudformation/src/configuration-manager.ts

```diff
@@ -920,7 +920,7 @@ async function determineAuthFlow(context: $TSContext, projectConfig?: ProjectCon
     const docsUrl = 'https://docs.amplify.aws/cli/usage/headless';
     context.print.error(errorMessage);
     context.print.info(`Access keys for continuous integration can be configured with headless parameters: ${chalk.green(docsUrl)}`);
-    await context.usageData.emitError(errorMessage);
+    await context.usageData.emitError(new Error(errorMessage));
```

Comment:
should this be `AmplifyError` or `AmplifyFault`?

Reply:
I'm not sure it matters here, let me check.

Reply:
The `emitError` shouldn't be called here ideally and handled with https://github.com/aws-amplify/amplify-cli/blob/dev/packages/amplify-cli/src/amplify-exception-handler.ts#L42 which is central place for error reporting. 

That might be outside the scope of this PR, but when that happens (i.e. throw error instead of emit), the `Error` **needs** to be an `AmplifyError` else it gets wrapped as `UnknownError` by the centralized error handling

Reply:
@Amplifiyer Perfect. I've got this logged in my doc for tracking additional tasks from this PR. Agreed that this is out of scope for this PR. I'll link you the doc.

Reply:
This looks like something I wrote a while back before `AmplifyError` existed. Ideally, this should throw an `AmplifyError` going forward for consistency and accurate error reporting.